### ISSUE TITLE
RFC: Provide API methods for Python 2 & 3

### ISSUE THREAD

**Issue by jalcine** [lambdalisue/vim-pyenv#24]
I use Python 3 predominately for my Neovim setup. However, I tend to work on Python 2 code and I'm interested if providing methods like `pyenv#pyenv#get_prog_for_version(version_number` would be something that you can see in the API for pyenv.

**Comment by lambdalisue** [comment]
Does https://github.com/lambdalisue/vim-pyenv/blob/master/autoload/pyenv/python.vim#L86-L112 help you?

**Comment by jalcine** [comment]
> Does https://github.com/lambdalisue/vim-pyenv/blob/master/autoload/pyenv/python.vim#L86-L112 help you?

No. I'm looking for something that if I had a `.pyenv-version` file of the following:

```
2.7.12
3.5.2
```

And I got something like the following for output:

```viml
:echo pyenv#python#get_major_version() == '3'
1 (True)
:echo pyenv#python#get_exec_for_current_version()
~/.anyenv/envs/pyenv/versions/2.7.12/bin/python
:echo pyenv#python#get_versions()
['2.7.12', '3.5.2']
```

This way, I could work with different versions of Python for different aspects of Vim.

**Comment by jalcine** [comment]
Provided time, I can work on something like this as a patch.
